HomeWork_03/DZ3.py

```diff
@@ -0,0 +1,79 @@
+#I love Python*42#
+
+# Zadanie 1
+x = ("I love Python")
+z = x*42
+print(z)
+
+#age_in_month#
+
+# Zadanie 2
+x = str("age_in_month")
+y = 37*12
+z = x+" "+str(y)
+print(z)
+
+#Take age in "y" Zadanie 2#
+
+# Zadanie 3
+i = str(" age_in_years")
+j = str(i)+" " + str(y//12)
+print (j)
+
+#----------------------#
+# Zadanie 4
+first_name = "Oleksandr"
+last_name = "Pitelin"
+full_name = first_name +" "+ last_name
+k = ("Му name is" +" "+ (full_name))
+l = ("I’m" + (j) + "years old")
+my_age =  k+" "+l
+
+print(my_age)
+
+#----------------------#
+# Zadanie 5
+
+a=5>7
+print(a)
+
+a=4<=4
+print(a)
+
+a=52//2<14*2
+print(a)
+
+a=51
+print(a>31 and a<55)
+
+a=12/3>2*7
+print(a)
+
+
+#insert (int) to  (str)#
+# Zadanie 6
+a=str(2)
```

Comment:
Initially, a b c should be integers. And you need to transform them to string when you create `d` variable